src/snmalloc/mem/corealloc.h

```diff
@@ -621,8 +621,7 @@ namespace snmalloc
     template<
       typename Config_ = Config,
       typename = std::enable_if_t<Config_::Options.CoreAllocOwnsLocalState>>
-    CoreAllocator(Range<capptr::bounds::Alloc>& spare, LocalCache* cache)
-    : attached_cache(cache)
+    CoreAllocator(Range<capptr::bounds::Alloc>& spare)
```

Comment:
This is going to leave `attached_cache` `nullptr` "more often" after construction (I guess you could always have passed in `nullptr` for the 2nd arg).  Are we guaranteed that `attach()` is called before `flush()` or before `handle_message_queue_inner()` decides to `post()`?